src/player.ts

```diff
@@ -229,7 +249,7 @@ export default class Player extends EventEmitter {
   // Not to confuse with the end of a round
   endTurn() {
     // Reset free terraforming steps
-    this.data.terraformSteps = 0;
+    this.data.terraformCostDiscount = 0;
```

Comment:
Actually this was my mistake, recently introduced. I thought it was `data.temporaryStep`. This line can be removed.